uspv/keyderiv.go

```diff
@@ -1,7 +1,7 @@
 package uspv
 
 import (
-	"fmt"
```

Comment:
Since keyderiv is interactive, these aren't suitable to be done via log.

It might make sense to remove key unwrapping from the lit UI, as it's the only interactive component left, and have unwrapping be done via RPC.  But that's another issue; for now we can leave this as interactive.